.editorconfig

```diff
@@ -13,9 +13,12 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
 
-[{.jshintrc,*.json,*.yml}]
+[*.yml]
 indent_style = space
 indent_size = 2
 
+[*.md]
+trim_trailing_whitespace = false
+
 [{*.txt,wp-config-sample.php}]
```

Comment:
We don't have a `wp-config-sample.php` file in this project, nor `.txt` files.

The `.txt` entry for WordPress core is because of some Windows incompatibility. I don't think we need that either.